security/README.md

```diff
@@ -0,0 +1,57 @@
+Security
+========
+
+A guide for practicing safe web.
+
+Think
+-----
+
+Security is important, and you can't practice these guidelines without
+understanding them. Make sure you understand each guideline, why it exists, and
+how to follow it.
+
+Failing to follow these guidelines will likely put you, your team, and your
+deployed services at risk of compromise or loss of privacy.
+
+Using Passwords
```

Comment:
Ah, I see.

I think I see this section as also basically mandatory. If you don't follow any of these guidelines, your accounts may become compromised. As your team probably has granted you access to privileged data, that means your team will suffer from your lack of personal security.

Basically, as a member of a developer team, you are given the keys to secure services and asked to keep those keys safe. If you reuse passwords, you're exposing the whole team to risk.